setup.py

```diff
@@ -30,7 +30,7 @@ def read(fname):
     },
     install_requires=[
         'json-rpc>=1.12,<2',
-        'dbt-core~=1.0.0b1'
+        'dbt-core>=1.0.0b1'
```

Comment:
@jtcohen6 does this seem right to you? The RPC server should work with any v1.* core version?

Reply:
Yes! I think let's just make this
```suggestion
        'dbt-core>=1'
```
(no b1) since it probably shouldn't install dbt-core prereleases by default. Users can still install dbt-core prereleases via
```
pip install --pre dbt-core dbt-rpc ...
```